In [57]:
import tweepy
import pandas as pd


api_key = '0n13FSWIknK0ZanB0Hsnju1xA'
api_key_secret = 'ztg0AlimkC1r1X1kW3CB2csmj5AhEvtQut8Xo4TggTYdwn3p63'
bearer_token = 'AAAAAAAAAAAAAAAAAAAAALlLnwEAAAAAgSeSd%2BDHxAB%2Bdhl%2BEblgcYhuCSs%3DcY9wgi40eYIJtUauHpTrGWq3W9sRjai40KE1dX5Hzpyu45rCnQ'
access_token = '1012017826566057984-rNjdVAqVaP3n8fXF4G0u5A8KxSDof1'
access_token_secret = 'NZhHtGqdoFGR2Q5tJh80ib85V3Cz7ElxfH4TBNYrTAvlF'

import tweepy

# Authenticate to Twitter
auth = tweepy.OAuthHandler(api_key, api_key_secret)
auth.set_access_token(access_token, access_token_secret)

# Create an API object
tweepy.API
api = tweepy.API(auth, wait_on_rate_limit=True)

# Get the top 10 trending topics in India
trends = api.get_place_trends(id=23424848)

# Print the top 10 trending topics
for value in trends[0]["trends"]:
    print(value["name"])



Forbidden: 403 Forbidden
453 - You currently have access to Twitter API v2 endpoints and limited v1.1 endpoints only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve

### Error while getting api acces

Error: Forbidden: 403 Forbidden
453 - You currently have access to Twitter API v2 endpoints and limited v1.1 endpoints only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve

1. Now this v1.1 cost '''100$''' so iam not able to pay that amount.
2. In v2 there is no access to use trends attribute.


## ALso this API connecting project is not taught in class